web/src/OverviewResourceBar.tsx

```diff
@@ -46,16 +55,259 @@ let ResourceStatus = styled.div`
   padding: 4px 16px;
 `
 
+type ResourceBarStatusProps = {
+  view: Proto.webviewView
+}
+
+// If there are more than 20 resources,
```

Comment:
Oh! You detailed an edge case that Surbhi and I hadn't actually considered in the design itself. I think of these as 1:1 with the resource list below, so each block always refers to a single specific resource. 

(After playing with this PR, I even had the thought that we could show a tooltip with the name of the resource on hover, so you could quickly see which resources were in an error state from this compact UI.)

I like what you did here for now, since it won't break on contact for users with tons of resources. But I made a note to discuss with Surbhi what the ideal design should be for 20+ resources!

Reply:
:+1: